cobigen-setup/files/cliExecutionCommands.md

```diff
@@ -0,0 +1,18 @@
+CobiGen CLI is installed inside your devonfw distribution. In order to execute it follow the next steps:
+
+1. Run console.bat, this will open a console. Here we will go inside the devonfw ide installation to execute the commands.

```

Comment:
Here I would write that the user has to open a terminal and navigate into the devonfw installation directory. .bat files won't work on Linux